src/UnicodePlots.jl

```diff
@@ -59,6 +68,8 @@ export GraphicsArea,
     spy,
     boxplot,
     boxplot!,
+    draw_axes!,
```

Comment:
I'm not sure if we really want to export this symbol. Do users have opportunities to call this themselves instead of to call this via, e.g., `isosurface!`?

Reply:
Yes, this was before I added the keyword `axes3d` for automating it. Will remove.